packages/gestalt/src/Avatar.js

```diff
@@ -33,8 +51,8 @@ const DefaultAvatar = ({ name }: { name: string }) => {
         >
           <title>{name}</title>
           <text
-            fontSize="40px"
-            fill="#111"
+            fontSize="50px"
```

Comment:
@nachofassini reverted back to `40px` - since that's what we did in the new design update #771